evm_loader/lib/src/account_storage.rs

```diff
@@ -1309,7 +1312,7 @@ impl<T: Rpc> SyncedAccountStorage for EmulatorAccountStorage<'_, T> {
         let signers = seeds
             .iter()
             .map(|s| {
-                let seed = s.iter().map(|s| s.as_slice()).collect::<Vec<_>>();
+                let seed = s.iter().map(std::vec::Vec::as_slice).collect::<Vec<_>>();
```

Comment:
There is an unnecessary qualification: `std::vec::Vec` can be just replaced with `Vec`.

Reply:
done